.gitignore

```diff
@@ -0,0 +1,2 @@
+dist/bundle.js
```

Comment:
If we ignore distilled folder, we will not have index.html available for deployment